fabric-rendering-v1/src/client/java/net/fabricmc/fabric/mixin/client/rendering/WorldRendererMixin.java

```diff
@@ -83,6 +83,19 @@ private void afterTerrainSetup(Camera camera, Frustum frustum, boolean hasForced
 		WorldRenderEvents.AFTER_SETUP.invoker().afterSetup(context);
 	}
 
+	@Inject(
+			method = "method_62214",
+			at = @At(
+				value = "INVOKE",
+				target = "Lnet/minecraft/util/profiler/Profiler;push(Ljava/lang/String;)V",
+				ordinal = 0,
+				shift = Shift.AFTER
+			)
+	)
```

Comment:
This points to `profiler.push("terrain");` if I'm not mistaken? Might be a good idea to note that with a comment since it's not obvious from the code.

Reply:
It does, I'll add a comment